src/app/components/sendForm/index.tsx

```diff
@@ -289,6 +253,17 @@ function SendForm({
     }
   }, [recepientError, associatedAddress]);
 
+  useEffect(() => {
```

Comment:
Currently, when you click "Cancel" on the "Review transaction" screen, even though you have the amount value set, you see "0 USD" under it. This code fixes this issue.